docs/step-by-step/kubeapps-on-tkg/step-2.md

```diff
@@ -0,0 +1,235 @@
+helm repo add bitnami https://charts.bitnami.com/bitnami
+
+# Create a 'kubeapps' namespace in our cluster
+kubectl create namespace kubeapps
+
+# Install a 'kubeapps' release in the 'kubeapps' namespace with the values defined at 'custom-values.yaml'
+helm install kubeapps --namespace kubeapps bitnami/kubeapps -f custom-values.yaml
+```
+
+At this point, Kubeapps is installed in the cluster.
+
+### Step 2.4: Configure Role-Based Access
+
+Once Kubeapps is installed, the next step is to configure access. Since Kubeapps delegates authorization to the existing Role-Based Access Control (RBAC) configured in the cluster, every permission should be granted using `ClusterRoleBinding` and `RoleBinding` objects. Please refer to the official documentation about [Kubeapps access control](https://github.com/kubeapps/kubeapps/blob/master/docs/user/access-control.md) for more information.
+
+> **NOTE**: RBAC configuration depends on your custom business requirements. The configuration shown below is only an example and not meant for production use. Please refer to the official [Kubernetes RBAC documentation](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) for more details.
+
+The configuration shown below demonstrates how to create a `ClusterRoleBinding` named `kubeapps-operator` with the `cluster-admin` role for a specified user. Replace the `EMAIL-ADDRESS` placeholder with the email address for the user, as specified in the OIDC provider and name the file `kubeapps-rbac.yaml`.
+
+```yaml
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kubeapps-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: EMAIL-ADDRESS
+```
+
+Apply this configuration by executing the following command:
+
+```bash
+kubectl apply -f kubeapps-rbac.yaml
+```
+
+At this point, the user having `EMAIL-ADDRESS` email account will have `cluster-admin` access and will be able to perform any desired action via the Kubeapps dashboard.
+
+### Step 2.5: Log in to Kubeapps with OIDC
+
+Once Kubeapps is installed and configured, the next step is to log in and access the Kubeapps Web dashboard. The procedure to do this depends on how Kubeapps was configured.
+
+1. If [the service was exposed externally](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps#exposing-externally), it may be accessed using a public IP address; if not, it can be accessed locally by forwarding the cluster port using the command below:
+
+    ```bash
+    kubectl port-forward -n kubeapps svc/kubeapps 8080:80
+    ```
+
+    This will start an HTTP proxy for secure access to the Kubeapps dashboard.
+
+2. Browse to [http://127.0.0.1:8080](http://127.0.0.1:8080) (when forwarding the port) or to the public IP address of the serevice (when exposing the service externally). You see the Kubeapps login page, as shown below:
```

Comment:
I might have missed it, but if they are using a public IP of the service, then the callback URI would need to be updated in the IdP (CSP) to include it?

Reply:
That's true, I should have mentioned it explicitly (I just mention _(this will be updated after Kubeapps is installed)_). I've added more details here. Thanks for pointing this out!